public/app/plugins/panel/geomap/layers/data/geojsonLayer.ts

```diff
@@ -108,7 +107,33 @@ export const geojsonLayer: MapLayerRegistryItem<GeoJSONMapperConfig> = {
           if (check.rule && !checkFeatureMatchesStyleRule(check.rule, feature)) {
             continue;
           }
-          return isPoint ? check.point : check.poly;
+
+          // Support dynamic values
+          if (check.state.fields) {
+            const values = { ...check.state.base };
+            for (const [key, val] of Object.entries(check.state.fields)) {
```

Comment:
For the immediate fix "support text" we could just do something like
```suggestion
           const {text} = check.state.fields;
            if(text) {
              values.text = `${feature.get(text)}`;
           }
```

We can likely hold of for the other dynamic types until we move the dataFrame flavor here also since we do not (yet) have a good way to get scaled values etc.